ci/pipeline.yml

```diff
@@ -458,30 +434,24 @@ resources:
     versioned_file: master-bosh.crt
 
 - name: common-development
-  type: cg-common
+  type: s3-iam
   source:
     bucket_name: {{private-bucket-development}}
-    secrets_passphrase: {{passphrase-development}}
-    secrets_file: nfs-development.yml
-    bosh_cert: bosh.pem
+    versioned_file: nfs-development-decrypted.yml
     region: {{aws-region}}
 
 - name: common-staging
-  type: cg-common
+  type: s3-iam
   source:
     bucket_name: {{private-bucket-staging}}
-    secrets_passphrase: {{passphrase-staging}}
-    secrets_file: nfs-staging.yml
-    bosh_cert: bosh.pem
+    versioned_file: nfs-staging-decrypted.yml
     region: {{aws-region}}
 
 - name: common-production
-  type: cg-common
+  type: s3-iam
   source:
     bucket_name: {{private-bucket-production}}
-    secrets_passphrase: {{passphrase-production}}
-    secrets_file: nfs-production.yml
-    bosh_cert: bosh.pem
+    versioned_file: nfs-production-decrypted.yml
     region: {{aws-region}}
```

Comment:
Looks like it's called `region_name` in the s3 resource: https://github.com/alphagov/paas-s3-resource#source-configuration

Reply:
Also, we should be able to drop `cg-common` from `resource_types`.